Where_Is_My_Wife/Assets/Hook_QTE/Scripts/HookUIBar.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using UnityEngine;
+using WhereIsMyWife.Controllers;
+using WhereIsMyWife.Managers;
+
+public interface IHookUIEvents
+{
+    Action<bool> QTEStateEvent { get; set; }
+    Action QTETimeExpired { get; set; }
+}
+
+public class HookUIBar : Singleton<HookUIBar>, IHookUIEvents
```

Comment:
All classes whould be in a namespace, in this case ´WhereIsMyWife.UI´ looks appropriate.

Reply:
902241d025ae61ebfb8e4b08abd978123a88dfcb